tests/python/pytorch/graphbolt/impl/test_ondisk_dataset.py

```diff
@@ -1191,6 +1191,345 @@ def test_OnDiskDataset_preprocess_homogeneous(edge_fmt):
         fused_csc_sampling_graph = None
 
 
+@pytest.mark.parametrize("edge_fmt", ["csv", "numpy"])
+def test_OnDiskDataset_preprocess_homogeneous_hardcode(edge_fmt):
+    """Test preprocess of OnDiskDataset."""
+    with tempfile.TemporaryDirectory() as test_dir:
+        # All metadata fields are specified.
+        dataset_name = "graphbolt_test"
+        num_nodes = 5
+        num_edges = 10
+        num_classes = 1
+
+        # Generate edges.
+        edges = np.array(
+            [[0, 0, 1, 1, 2, 2, 3, 3, 4, 4], [1, 2, 2, 3, 3, 4, 4, 0, 0, 1]]
```

Comment:
show the graph in COO format like this for better readability. https://github.com/dmlc/dgl/blob/af870387fc4a0b9b67c24fd49e5f68e133c71273/tests/python/pytorch/graphbolt/impl/test_fused_csc_sampling_graph.py#L242-L247

Reply:
and paste the node/edge feat as well.